config/config.txt

```diff
@@ -541,3 +541,9 @@ DEFAULT_VIEW_SQUARE 15x15
 
 ## Add the ID of the role you want assigning here
 #DISCORD_ROLEID 000000000000000000
+
+##Amount of time required for senior job titles in minutes
+SENIOR_TIMELOCK 3000
+
+##Amount of time required for ULTRA senior job titles in minutes
+SENIOR_TIMELOCK 30000
```

Comment:
i really dislike gatekeeping. 500 hours in a single job? My most played job is 300

Reply:
It's a config option for a reason.